translations/ui-orders/en.json

```diff
@@ -402,9 +402,9 @@
   "linesLimit.reasonText": "This would exceed the maximum number of purchase order lines permitted by system settings.",
   "location.button.addLocation": "Add Location",
   "location.nameCode": "Name (Code)",
-  "location.quantityElectronic.exceeds": "Quantity Electronic exceeds Cost Details quantity",
+  "location.quantityElectronic.not.match": "PO Line electronic quantity and Locations electronic quantity do not match",
   "location.quantityElectronic": "Quantity Electronic",
-  "location.quantityPhysical.exceeds": "Quantity Physical exceeds Cost Details quantity",
+  "location.quantityPhysical.not.match": "PO Line physical quantity and Locations physical quantity do not match",
```

Comment:
due to established pattern it should be `location.quantityPhysical.notMatch` IMHO

Reply:
fixed